conf/uzl_omics.config

```diff
@@ -0,0 +1,25 @@
+params {
+    config_profile_description = 'University of Luebeck, OMICS cluster profile provided by nf-core/configs.'
+    config_profile_contact = 'Finja Sygo (@finjasygo) and Varun Sreenivasan (@varunkas) (Institut fuer Humangenetik)'
+    config_profile_url = 'https://www.itsc.uni-luebeck.de/dienstleistungen/omics.html'
+}
+
+params {
+    max_memory = 760.GB    
+    max_cpus   = 48
+    max_time   = 72.h     
+}
+
+process {
+    executor = 'slurm'
+    queue = 'shortterm'
+    scratch = '$SCRATCH'
+}
+
+workDir = "$WORK/nf-core_rnaseq_workdir"
+
+singularity {
+    enabled = true
+    runOptions = "-B $WORK/nf-core_rnaseq_workdir -B $HOME/.nextflow/assets/nf-core/rnaseq/bin"
```

Comment:
These are very specific for an institutional configuration.

My recommendation (for singularity) is to have the configuration bind all "project" file spaces by default, but for these, do you need more than `autoMounts = true`?